src/main/java/com/yelp/nrtsearch/server/luceneserver/SearchHandler.java

```diff
@@ -763,7 +768,6 @@ private CompositeFieldValue getFieldForHit(
 
       // We detect invalid field above:
       assert fd != null;
-
       if (fd instanceof VirtualFieldDef) {
```

Comment:
Does the runtime field also need to be covered in this code path?

Reply:
Somehow the code never comes here when I put a breakpoint in this line. I assume it's not necessary to do that anymore. It doesn't stop here, even when I have a virtual field in  the request.

Reply:
Do you know how I can test that locally?

Reply:
This is the config used for the current testing https://github.com/Yelp/nrtsearch/blob/master/src/test/java/com/yelp/nrtsearch/server/grpc/MultiSegmentParallelFieldsFetchTest.java#L25

Reply:
Thanks. I was able to debug this part of the code with the above configs. It now returns the same object as the one with fetching docs.